modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -452,6 +449,23 @@ describe('TSS Ecdsa Utils:', async function () {
 
       sandbox.verifyAndRestore();
     });
+
+    async function setupSignTxRequest(isTxRequest = true) {
```

Comment:
we could rename may to the below suggestion to give more clarity on what the function does
```suggestion
    async function setupSignTxRequestMocks(isTxRequest = true) {
```